### PR TITLE
Paper review

### LINKED ISSUE #226: JOSS Software paper issues
A few notes on the JOSS software paper: 

1. For the checklist item: State of the field: Do the authors describe how this software compares to other commonly-used packages?

This could be improved. For example, what are the existing approaches to interact with data from secuTrial? How does the functionality in secuTrialR compare with that of R interfaces to competing clinical data management systems such as redcapR and ox (openclinica)? 

2. Issues in the references: 

- Kelion 2020 -- Microsoft and COVID-19 should be capitalized
- Kuchinke -- Europe and ECRIN
- Merlot -- Url runs off page
- Rstudio and Wickham -- R should be capitalized

### PR DESCRIPTION
**Pull request by aghaynes**
closes #226

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -68,16 +68,25 @@ spent on data management should be as small as possible. The use of `secuTrialR`
 necessary for data management, enables swift quantitative analyses through preimplemented functionalities and, most importantly,
 standardizes the interaction with data exports from secuTrial, thus allowing robust and reproducible science.
 
+While some CDMSs provide APIs (e.g. REDCap [@Harris2009; @Harris2019]) or ODBC connections (e.g.
```
Comment (PatrickRWright) [inline paper/paper.md:71]: Rather just CDMS?

What is an ODBC? Maybe write the abbreviation out.

File: paper/paper.md [hunk 2]
```diff
@@ -68,16 +68,25 @@ spent on data management should be as small as possible. The use of `secuTrialR`
 necessary for data management, enables swift quantitative analyses through preimplemented functionalities and, most importantly,
 standardizes the interaction with data exports from secuTrial, thus allowing robust and reproducible science.
 
+While some CDMSs provide APIs (e.g. REDCap [@Harris2009; @Harris2019]) or ODBC connections (e.g. 
+[2mt's WebSpirit](http://www.2mt-software.de)) to download data easily, using SecuTrial's SOAP API involves querying
```
Comment (PatrickRWright) [inline paper/paper.md:72]: secuTrial not capital

File: paper/paper.md [hunk 3]
```diff
@@ -68,16 +68,25 @@ spent on data management should be as small as possible. The use of `secuTrialR`
...
+individual datapoints. This results in an extraordinarily high number of 
+queries even to download a relatively small database, and high demand on servers. As such, approaches such as those 
+for REDCap (e.g. the [REDCapR](https://CRAN.R-project.org/package=REDCapR) package which can interface to REDCaps REST
```
Comment (PatrickRWright) [inline paper/paper.md:75]: REDCap's

File: paper/paper.md [hunk 4]
```diff
@@ -68,16 +68,25 @@ spent on data management should be as small as possible. The use of `secuTrialR`
 necessary for data management, enables swift quantitative analyses through preimplemented functionalities and, most importantly,
 standardizes the interaction with data exports from secuTrial, thus allowing robust and reproducible science.
 
+While some CDMSs provide APIs (e.g. REDCap [@Harris2009; @Harris2019]) or ODBC connections (e.g. 
+[2mt's WebSpirit](http://www.2mt-software.de)) to download data easily, using SecuTrial's SOAP API involves querying
+individual datapoints. This results in an extraordinarily high number of 
+queries even to download a relatively small database, and high demand on servers. As such, approaches such as those 
+for REDCap (e.g. the [REDCapR](https://CRAN.R-project.org/package=REDCapR) package which can interface to REDCaps REST 
+API and download all data in a single query, but does no data preparation) are not suitable for SecuTrial.
```
Comment (PatrickRWright) [inline paper/paper.md:76]: secuTrial

File: paper/paper.md [hunk 5]
```diff
@@ -68,16 +68,25 @@ spent on data management should be as small as possible. The use of `secuTrialR`
 necessary for data management, enables swift quantitative analyses through preimplemented functionalities and, most importantly,
 standardizes the interaction with data exports from secuTrial, thus allowing robust and reproducible science.
 
+While some CDMSs provide APIs (e.g. REDCap [@Harris2009; @Harris2019]) or ODBC connections (e.g. 
+[2mt's WebSpirit](http://www.2mt-software.de)) to download data easily, using SecuTrial's SOAP API involves querying
+individual datapoints. This results in an extraordinarily high number of 
+queries even to download a relatively small database, and high demand on servers. As such, approaches such as those 
+for REDCap (e.g. the [REDCapR](https://CRAN.R-project.org/package=REDCapR) package which can interface to REDCaps REST 
+API and download all data in a single query, but does no data preparation) are not suitable for SecuTrial. 
+Another approach is to parse data exported manually from websites (e.g. the [ox](https://github.com/acobos/ox) package for importing [OpenClinica](https://www.openclinica.com) exports into R). This latter approach is used in `secuTrialR`.
```
Comment (PatrickRWright) [inline paper/paper.md:77]: This approach is also used in `secuTrialR`.

**Review by PatrickRWright — CHANGES_REQUESTED** [review]
Just a few small things.

Thank you @aghaynes

File: paper/paper.md (hunk 1 above)
Comment (aghaynes) [reply paper/paper.md]: done

File: paper/paper.md (hunk 2 above)
Comment (aghaynes) [reply paper/paper.md]: done

File: paper/paper.md (hunk 3 above)
Comment (aghaynes) [reply paper/paper.md]: done

File: paper/paper.md (hunk 4 above)
Comment (aghaynes) [reply paper/paper.md]: done

File: paper/paper.md (hunk 5 above)
Comment (aghaynes) [reply paper/paper.md:77]: see the last sentence... ;)

Comment (PatrickRWright) [reply paper/paper.md:77]: The suggestion was to omit the word "latter". Do you agree? If you don't we can leave it. Sounds better to me without "latter".

Comment (aghaynes) [reply paper/paper.md:77]: ah... thats what you mean... fine with me

Comment (aghaynes) [reply paper/paper.md]: done

**Pull request merged**